packages/connect-webextension/src/proxy/index.ts

```diff
@@ -0,0 +1,134 @@
+import EventEmitter from 'events';
+
+// NOTE: @trezor/connect part is intentionally not imported from the index due to NormalReplacementPlugin
+// in packages/suite-build/configs/web.webpack.config.ts
+import {
+    ERRORS,
+    IFRAME,
+    POPUP,
+    createErrorMessage,
+    ConnectSettings,
+    Manifest,
+    CallMethod,
+} from '@trezor/connect/lib/exports';
+import { factory } from '@trezor/connect/lib/factory';
+
+import { parseConnectSettings } from '../connectSettings';
+import { WindowServiceWorkerChannel } from '../channels/window-serviceworker';
+
+const eventEmitter = new EventEmitter();
+let _settings = parseConnectSettings();
+let _channel: any;
+
+const manifest = (data: Manifest) => {
+    _settings = parseConnectSettings({
+        ..._settings,
+        manifest: data,
+    });
+};
+
+const dispose = () => {
+    eventEmitter.removeAllListeners();
+    _settings = parseConnectSettings();
+
+    return Promise.resolve(undefined);
+};
+
+const cancel = () => {
+    if (_channel) {
+        _channel.clear();
+    }
+};
+
+const init = (settings: Partial<ConnectSettings> = {}): Promise<void> => {
+    _settings = parseConnectSettings({ ..._settings, ...settings });
+
+    if (!_settings.manifest) {
+        throw ERRORS.TypedError('Init_ManifestMissing');
+    }
+
+    if (!_settings.transports?.length) {
+        _settings.transports = ['BridgeTransport', 'WebUsbTransport'];
+    }
```

Comment:
these lines are duplicated with `packages/connect-webextension/src/index.ts` proxy should not implemnet any logic. it should only return data from service worker.

Reply:
Removed, along with all the references to settings there https://github.com/trezor/trezor-suite/pull/10751/commits/c294594cbcbc44bc391611dccb010090434d9de3